roles/kubevirt-resources/tasks/provision.yaml

```diff
@@ -0,0 +1,35 @@
+- name: Login As Super User
+  command: "oc login -u {{ admin_user }} -p {{ admin_password }}"
+  when: cluster=="openshift"
+        and admin_user is not undefined
+        and admin_password is not undefined
+
+- name: Create {{ namespace }} namespace
+  shell: kubectl create namespace {{ namespace }}
+  ignore_errors : true
+
+- name: Add Privileged Policy
+  command: "oc adm policy add-scc-to-user privileged -z {{ item }} -n {{ namespace }}"
+  with_items:
+    - kubevirt-privileged
+    - kubevirt-controller
+    - kubevirt-infra  # For KubeVirt v0.2.0
+  when: cluster=="openshift"
+
+- name: Add Hostmount-anyuid Policy
+  command: "oc adm policy add-scc-to-user hostmount-anyuid -z kubevirt-infra -n {{ namespace }}"
+  when: cluster=="openshift"
+
+- name: Download Kubevirt Template
+  get_url:
+    url: "https://raw.githubusercontent.com/kubevirt/kubevirt/master/manifests/release/kubevirt.yaml.in"
+    dest: /tmp/kubevirt.yaml.j2
```

Comment:
1. I think that the version of the manifests should be configurable, so it can be matched to the version of kubevirt's 
    containers.

2. If this role is going to be used in the APB, I don't think that we should dynamically fetch the manifest, rather include it in the APB. When running the APB, we shouldn't assume that it has internet access (kubevirt's containers can be fetched from a private registry).